.eslintrc.json

```diff
@@ -36,7 +36,7 @@
     }
   },
   "rules": {
-    "import/no-default-export": "error",
+    "import/no-default-export": "off",
```

Comment:
eh I'm pretty in favor of not using default exports. What's the thought here?

Reply:
Just that we were ignoring it in a lot of the files that I was working on and it doesn't seem necessary - why prevent default exports? I'm fine to revert this tbh I don't feel very strongly